eos/b-decays/b-to-vec-l-nu.cc

```diff
@@ -408,87 +408,87 @@ namespace eos
     }
 
     double
-    BToVectorLeptonNeutrino::differential_J1c_normalized(const double & q2) const
+    BToVectorLeptonNeutrino::differential_J1c(const double & q2) const
     {
         auto   o = _imp->differential_angular_observables(q2);
-        return 3.0 / 4.0 * o.vv10() / o.normalized_decay_width();
+        return 3.0 / 4.0 * o.vv10();
     }
 
     double
-    BToVectorLeptonNeutrino::differential_J1s_normalized(const double & q2) const
+    BToVectorLeptonNeutrino::differential_J1s(const double & q2) const
     {
         auto   o = _imp->differential_angular_observables(q2);
-        return 3.0 / 4.0 * o.vv1T() / o.normalized_decay_width();
+        return 3.0 / 4.0 * o.vv1T();
     }
 
     double
-    BToVectorLeptonNeutrino::differential_J2c_normalized(const double & q2) const
+    BToVectorLeptonNeutrino::differential_J2c(const double & q2) const
     {
         auto   o = _imp->differential_angular_observables(q2);
-        return 3.0 / 4.0 * o.vv20() / o.normalized_decay_width();
+        return 3.0 / 4.0 * o.vv20();
     }
 
     double
-    BToVectorLeptonNeutrino::differential_J2s_normalized(const double & q2) const
+    BToVectorLeptonNeutrino::differential_J2s(const double & q2) const
     {
         auto   o = _imp->differential_angular_observables(q2);
-        return 3.0 / 4.0 * o.vv2T() / o.normalized_decay_width();
+        return 3.0 / 4.0 * o.vv2T();
     }
 
     double
-    BToVectorLeptonNeutrino::differential_J3_normalized(const double & q2) const
+    BToVectorLeptonNeutrino::differential_J3(const double & q2) const
     {
         auto   o = _imp->differential_angular_observables(q2);
-        return 3.0 / 4.0 * o.vv4T() / o.normalized_decay_width();
+        return 3.0 / 4.0 * o.vv4T();
     }
 
     double
-    BToVectorLeptonNeutrino::differential_J4_normalized(const double & q2) const
+    BToVectorLeptonNeutrino::differential_J4(const double & q2) const
     {
         auto   o = _imp->differential_angular_observables(q2);
-        return 3.0 / 4.0 * -o.vv10T() / o.normalized_decay_width();
+        return 3.0 / 4.0 * -o.vv10T();
```

Comment:
We need to check whether there should be these minus signs in

J_4, 6s, 6c, 7, 9

Reply:
Check, or did you come to a consensus with @MartinJung1978 that this is not conforming to the Belle definitions?

Reply:
We checked that this is not conforming with the Belle 2018 definition. You might cherry-pick from my git-repo (branch BtoDstarlnu) latest commit for correction of minus signs and adaption of test cases.

Reply:
I try to finish another commit with a test case for B->D* lv angular observables including new physics contributions that could be also added to this pull request.

Reply:
Code changes look fine for me.